app/controllers/api/issues_controller.rb

```diff
@@ -126,9 +126,20 @@ def archive_issue
     end
 
     def assign_issue
-      user, repo, number, @assignee = params[:user], params[:repo], params[:number], params[:assignee]
-      @issue = huboard.board(user, repo).issue(number)
-        .patch 'assignee' => @assignee
+      user, repo, number, assignees = params[:user], params[:repo], params[:number], params[:assignees]
+      issue = huboard.board(user, repo).issue(number)
+      @issue = issue.add_assignees(assignees) if issue['assignees']
+      @issue = issue.patch('assignee' => assignees.first) if !issue['assignees'] 
```

Comment:
Maybe not idiomatic Ruby, but I'd prefer an explicit `if ... else ...` block rather than a pair of statements with inverted conditions. Seems like it would be easier to see exactly what's happening, since we need to do one or the other.